src/pymorize/ssh_tunnel.py

```diff
@@ -0,0 +1,40 @@
+import subprocess
+
+import rich_click as click
+
+
+@click.command()
+@click.option("--local-port", default=8787, help="Local port to forward from")
+@click.option("--remote-port", default=8787, help="Remote port to forward to")
+@click.option("--gateway", default="levante2.dkrz.de", help="Gateway server hostname")
+@click.option("--compute-node", required=True, help="Compute node hostname")
+@click.option("--username", required=True, help="Username for SSH connections")
+def ssh_tunnel_cli(local_port, remote_port, gateway, compute_node, username):
+    """
+    Create an SSH tunnel to access a Dask dashboard on a remote compute node.
+
+    This script sets up an SSH tunnel to access a Dask dashboard running
+    on a remote compute node, routed through a gateway server.
+    """
+    ssh_command = (
+        f"ssh -nNT -L {local_port}:{compute_node}:{remote_port} {username}@{gateway}"
```

Comment:
###  how about using ControlMaster and ControlPath while creating the connection
"ssh -nNT -M -S tmp_tunnel -L {local_port}:{compute_node}:{remote_port} {username}@{gateway}"

Reply:
I guess it will close the ports gracefully. I am not 100% sure but could give a try.

Reply:
The ports close regularly after a Control + C, so I don't think we need to complicate the SSH command too much by using this.